src/components/Header.js

```diff
@@ -1,46 +1,49 @@
 import React, { Component } from 'react';
 import { Localized } from 'fluent-react/compat';
+import { Navbar, Nav, NavItem, NavDropdown, MenuItem } from 'react-bootstrap';
```

Comment:
NavDropdown and MenuItem seem to be unused, can you remove them from the import?

Reply:
also, could you add react-bootstrap to the dependencies in package.json? You can rerun `npm install --save react-bootstrap" and it should do it automatically.

Reply:
Fixed on https://github.com/kelimuttu/activate-rewrite/commit/10802b352c83e6d7a3378a6cde7dee133c13968b